sdk/containerregistry/Azure.Containers.ContainerRegistry/src/autorest.md

```diff
@@ -5,8 +5,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 ``` yaml
 title: Container Registry
 input-file:
- - https://github.com/Azure/azure-rest-api-specs/blob/c8d9a26a2857828e095903efa72512cf3a76c15d/specification/containerregistry/data-plane/Azure.ContainerRegistry/stable/2021-07-01/containerregistry.json
- 
+ - $(this-folder)/../../../../../swagger_azure-rest-api-specs/specification/containerregistry/data-plane/Azure.ContainerRegistry/stable/2021-07-01/containerregistry.json
```

Comment:
Is this file checked-in?

Reply:
🤦🏻‍♀️

I was using it to debug and pushed too fast.  Thanks for catching, and will fix!